client/src/components/EmailRecipientCombobox.vue

```diff
@@ -11,7 +13,7 @@ export default {
       type: Array,
     },
     candidates: {
-      type: Array,
+      type: Array as PropType<Array<string>>,
```

Comment:
```suggestion
      type: Array as PropType<string[]>,
```
I think `type[]` is generally preferred to `Array<type>`, although I personally do not feel strongly on the matter.